converters/google/resources/resource_converters.go

```diff
@@ -1,14 +1,14 @@
 // ----------------------------------------------------------------------------
 //
-//     ***     AUTO GENERATED CODE    ***    Type: MMv1     ***
+//	***     AUTO GENERATED CODE    ***    Type: MMv1     ***
 //
 // ----------------------------------------------------------------------------
 //
-//     This file is automatically generated by Magic Modules and manual
-//     changes will be clobbered when the file is regenerated.
+//	This file is automatically generated by Magic Modules and manual
+//	changes will be clobbered when the file is regenerated.
 //
-//     Please read more about how to change this file in
-//     .github/CONTRIBUTING.md.
+//	Please read more about how to change this file in
+//	.github/CONTRIBUTING.md.
```

Comment:
These whitespace changes will get overwritten - could you revert them to make the diffs simpler?